distributed/comm/tcp.py

```diff
@@ -137,7 +137,7 @@ def convert_stream_closed_error(obj, exc):
         # The stream was closed because of an underlying OS error
         exc = exc.real_error
         if isinstance(exc, ssl.SSLError):
-            if "UNKNOWN_CA" in exc.reason:
+            if exc.reason and "UNKNOWN_CA" in exc.reason:
```

Comment:
Note: It looks like the typing information for `typeshed` is off: https://github.com/python/typeshed/blob/edc0ecd8579389cc6bb75b343e1001424009c09b/stdlib/ssl.pyi#L33

Reply:
Could you raise the issue upstream?